airflow/models/__init__.py

```diff
@@ -4293,6 +4413,7 @@ class DagRun(Base, LoggingMixin):
     run_id = Column(String(ID_LEN))
     external_trigger = Column(Boolean, default=True)
     conf = Column(PickleType)
+    graph_id = Column(Integer, default=0)
```

Comment:
I'd rather this was nullable than have a default of 0

Reply:
already fix this